Jenkinsfile

```diff
@@ -56,7 +60,7 @@ pipeline {
                                 choiceParam('BUILD_TYPE', ["Static", "Release", "Component", "Debug"])
                                 booleanParam('WIPE_WORKSPACE', false)
                                 booleanParam('USE_GOMA', true)
-                                booleanParam('SKIP_SIGNING', true)
+                                booleanParam('SKIP_SIGNING', ${SKIP_SIGNING_DEFAULT})
```

Comment:
```groovy
booleanParam('SKIP_SIGNING', SKIP_SIGNING_DEFAULT)
```

Reply:
Regarding your second suggestion of keeping the default in devops: Is this feasible given that there is a boolean `SKIP_SIGNING` parameter here in `Jenkinsfile`? With your suggestion, what should the behavior be if that parameter is `true`/`false`?

I had a similar thought to you and my best idea would have been to make `SKIP_SIGNING` a tristate variable (true/false/undefined). If true/false, that gets passed to the child job. If undefined, the child job's value takes preference, which in the case of Windows would now be false. But I'm not sure it's possible to have a `booleanParam` (or other param) that's such a tristate.

Reply:
> Are you sure `SKIP_SIGNING_DEFAULT` instead of `${SKIP_SIGNING_DEFAULT}` will work @wknapik? We're inside a string here.

Ah, you're right. I missed that fact that it's a string. So we do need `${SKIP_SIGNING_DEFAULT}`.

Reply:
I appreciate the suggestion. I also like to improve as I go when feasible. My problem is that I fail to see how your suggestion can work. There's a `SKIP_SIGNING` parameter in `Jenkinsfile` that's used for all OSs. If that parameter stays (and that assumption is where I may misunderstand your suggestion), then it would be weird to respect it for the other OSs, but to essentially ignore it for Windows. And I tried to explain above that I believe the fact that it's a boolean parameter means that we could, for Windows, ignore it either always or never, but not selectively depending on whether the caller of the parent PR has made an explicit choice.

Reply:
What I'm talking about is some way dropping this entire code block https://github.com/brave/brave-core/blob/ace27b1ed20ddd91dc3c528712d6f37740113108/Jenkinsfile#L54,L69, or at least dropping select parameters from it, so that default values from the devops repo would apply. I we can have that - great, if we can't - let's merge as is.

Reply:
> What I'm talking about is some way dropping this entire code block

I understood that.

> or at least dropping select parameters from it, so that default values from the devops repo would apply.

My point was that we have a value for `SKIP_SIGNING` here, in this Jenkinsfile. This value can be `true` or `false` and I expect that we need to respect it. But if that's the case, then we can never defer to the default of the child job. Because the value from the parent (whether set explicitly, or by the default, which we cannot distinguish) will always overrule the child.

I'll merge as-is.